queue/group_remote_mongo.go

```diff
@@ -59,11 +59,11 @@ type MongoDBQueueGroupOptions struct {
 
 func (o MongoDBQueueGroupOptions) validate() error {
 	catcher := grip.NewBasicCatcher()
-	catcher.NewWhen(o.TTL < 0, "TTL must be greater than or equal to 0")
-	catcher.NewWhen(o.TTL > 0 && o.TTL < time.Second, "TTL cannot be less than 1 second, unless it is 0")
-	catcher.NewWhen(o.PruneFrequency < 0, "prune frequency must be greater than or equal to 0")
-	catcher.NewWhen(o.PruneFrequency > 0 && o.TTL < time.Second, "prune frequency cannot be less than 1 second, unless it is 0")
-	catcher.NewWhen((o.TTL == 0 && o.PruneFrequency != 0) || (o.TTL != 0 && o.PruneFrequency == 0), "ttl and prune frequency must both be 0 or both be not 0")
+	catcher.NewWhen(o.TTL < 0, "TTL cannot be negative")
+	catcher.NewWhen(o.TTL > 0 && o.TTL < time.Second, "TTL cannot be less than 1 second, unless it is not set")
+	catcher.NewWhen(o.PruneFrequency < 0, "prune frequency cannot be negative")
+	catcher.NewWhen(o.PruneFrequency > 0 && o.TTL < time.Second, "prune frequency cannot be less than 1 second, unless it is not set")
+	catcher.NewWhen((o.TTL == 0 && o.PruneFrequency != 0) || (o.TTL != 0 && o.PruneFrequency == 0), "TTL and prune frequency must both be set or both be not set")
```

Comment:
`must specify TTL and prune frequency or neither`

Reply:
Done.